package.json

```diff
@@ -44,7 +44,7 @@
     "@types/swagger-jsdoc": "^6.0.4",
     "axios": "^1.7.7",
     "cors": "^2.8.5",
-    "discord-rpc": "^4.0.1",
+    "discord-rpc": "github:sKiLdUsT/RPC#types",
```

Comment:
This should probably be published to NPM to avoid relying on a github repo which could vanish anytime

Reply:
discordjs/RPC has not been updated in 3 years, see [this issue](https://github.com/discordjs/RPC/pull/149)
Mentioned in that issue is [this PR](https://github.com/Inrixia/neptune-plugins/pull/37) that uses [@xhayper/discord-rpc](https://github.com/xhayper/discord-rpc).
I'll rewrite it to use that tomorrow.